NEWS.md

```diff
@@ -1,4 +1,21 @@
-## 1.2.0 - Unreleased
+## 2.1.0 - Unreleased
+
+## 2.0.0 - Released
+This release contains implementation of `encumbrance` and `fund` endpoints, finance schemas updates, created 
+unique index on fund `code` column.
+
+[Full Changelog](https://github.com/folio-org/mod-finance-storage/compare/v1.1.0...v2.0.0)
+
+### Stories
+* [MODFISTO-8](https://issues.folio.org/browse/MODFISTO-8) Fund schema updated
```

Comment:
1. Links to JIRA stories/bugs in descending order (highest to lowest)
2. Tabs and spaces are mixed

Reply:
fixed